reaktive-testing/src/commonMain/kotlin/com/badoo/reaktive/test/maybe/TestMaybe.kt

```diff
@@ -4,9 +4,14 @@ import com.badoo.reaktive.maybe.Maybe
 import com.badoo.reaktive.maybe.MaybeCallbacks
 import com.badoo.reaktive.maybe.MaybeObserver
 import com.badoo.reaktive.test.base.TestSource
+import com.badoo.reaktive.utils.freeze
 
 class TestMaybe<T> : TestSource<MaybeObserver<T>>(), Maybe<T>, MaybeCallbacks<T> {
 
+    init {
+        freeze()
```

Comment:
Maybe move it into `TestSource` to avoid duplication?

Reply:
It will work for now, but if you will add a field later into descendant class it will crash, because the class is already frozen. What do you think?

Reply:
And it won't work for TestObserver as we already have fields in descendants.

Reply:
Then keep it as it is now.